iteration/iteration.go

```diff
@@ -196,10 +200,21 @@ func (m *GormIterationRepository) Save(ctx context.Context, i Iteration) (*Itera
 	return &i, nil
 }
 
-// CanStartIteration checks the rule - Only one iteration from a space can have state=start at a time.
+// CanStartIteration checks the rule -
+// 1. Only one iteration from a space can have state=start at a time.
+// 2. Root iteration of the space can not be started.(Hence can not be closed - via UI)
+// Currently there is no State-machine for state transitions of iteraitons
+// till then we will not allow user to start root iteration.
 // More rules can be added as needed in this function
 func (m *GormIterationRepository) CanStartIteration(ctx context.Context, i *Iteration) (bool, error) {
```

Comment:
Done in https://github.com/almighty/almighty-core/pull/990/commits/924c77e78547dec4eaa0b639b8af468e07e12cb4